checker/src/org/checkerframework/checker/index/upperbound/UpperBoundAnnotatedTypeFactory.java

```diff
@@ -133,6 +129,14 @@ private SearchIndexAnnotatedTypeFactory getSearchIndexAnnotatedTypeFactory() {
     }
 
     /**
+     * Provides a way to query the IndexOf Checker, which helps the Index Checker type the results
```

Comment:
Throughout, comments like this don't help me much.  It might be better to say what it represents.